runtime-sdk/src/modules/rofl/types.rs

```diff
@@ -109,3 +109,9 @@ pub struct AppInstanceQuery {
     /// Runtime Attestation Key.
     pub rak: PublicKey,
 }
+
+/// Minimum stake thresholds for managing ROFL application.
```

Comment:
```suggestion
/// Stake thresholds for managing ROFL application.
```

Optionally you could also generalize a bit more and rename to `StakeThresholds`, change query to `rofl.StakeThresholds` and field `create` to `app_create`.